source/js/components/news/news.jsx

```diff
@@ -49,9 +49,6 @@ export default class News extends React.Component {
         <div key={newsItem.headline}>
           <div className="mb-3 news-item">
             <div className="d-flex align-items-center mb-2">
-              {newsItem.glyph && (
```

Comment:
we forgot to remove this as part of https://github.com/mozilla/foundation.mozilla.org/commit/6ea880a744fae9af4ba7644128fac5c18b0741ef#diff-d710f90370f4cafe437456801d91d464